sdk/docs/core/README.md

```diff
@@ -85,40 +85,46 @@ As our SDK performs operations, it can send log messages to a customer-defined c
 To enable logging, you must first write a callback function that our logging mechanism will invoke periodically with messages. The function signature must match this type definition (defined in the [az_log.h](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/inc/azure/core/az_log.h) file):
 
    ```C
-   typedef void (*az_log_fn)(az_log_classification classification, az_span message);
+   typedef void (*az_log_message_fn)(az_log_classification classification, az_span message);
```

Comment:
You need to mention this in changelog.

Reply:
This PR isn't making this change. Will do it separately.